server/services/nextcloud-talk/lib/message.bot.js

```diff
@@ -0,0 +1,76 @@
+const EventEmitter = require('events');
+const get = require('get-value');
+const logger = require('../../../utils/logger');
+
+class NextcloudTalkBot extends EventEmitter {
+  constructor(gladys, serviceId, token) {
+    super();
+    this.gladys = gladys;
+    this.serviceId = serviceId;
+    this.token = token;
+    this.isPolling = false;
+  }
+
+  async startPolling() {
```

Comment:
We usually extract class methods in single files.

Reply:
I think it's a bit particular, we will need to handle multiple bots in the same time, one for each user and they can have different status. If you have a better way to do it we can still change it